keps/sig-instrumentation/1668-log-tracking/README.md

```diff
@@ -268,232 +130,108 @@ OpenTelemetry has SpanContext which is used for propagation of K8s component.
-- N installs
-- More rigorous forms of testing—e.g., downgrade tests and scalability tests
-- Allowing time for feedback
+### Design of Mutating webhook(Out of tree)
+**Extract SpanContext and initial-trace-id from request's header**
 
-**Note:** Generally we also wait at least two releases between beta and
-GA/stable, because there's no opportunity for user feedback, or even bug reports,
-in back-to-back releases.
+**Update SpanContext and initial-trace-id to object**
 
-#### Removing a Deprecated Flag
+- if there is initial-trace-id, add trace-id, span-id and initial-trace-id to annotation (This is the case for requests from controller.)
 
-- Announce deprecation and support policy of the existing flag
-- Two versions passed since introducing the functionality that deprecates the flag (to address version skew)
-- Address feedback on usage/changed behavior, provided on GitHub issues
-- Deprecate the flag
+- if there is no initial-trace-id, check the request's operation
+  - if operation is create, copy the trace-id as initial-trace-id, and add trace-id, span-id and initial-trace-id to annotation (This is the case for requests from kubectl create.)
+  - if operation is not create, add trace-id, span-id to annotation (This is the case for requests from kubectl other than create.)
 
-**For non-optional features moving to GA, the graduation criteria must include 
-[conformance tests].**
+**Persist object to etcd**
 
-[conformance tests]: https://git.k8s.io/community/contributors/devel/sig-architecture/conformance-tests.md
--->
+### Risks and Mitigations
 
 TBD
 
-### Upgrade / Downgrade Strategy
-
-<!--
-If applicable, how will the component be upgraded and downgraded? Make sure
-this is in the test plan.
-
-Consider the following in developing an upgrade/downgrade strategy for this
-enhancement:
-- What changes (in invocations, configurations, API use, etc.) is an existing
-  cluster required to make on upgrade, in order to maintain previous behavior?
-- What changes (in invocations, configurations, API use, etc.) is an existing
-  cluster required to make on upgrade, in order to make use of the enhancement?
--->
-TBD
+### Test Plan
 
-### Version Skew Strategy
+All added code will be covered by unit tests.
 
-<!--
-If applicable, how will the component handle version skew with other
-components? What are the guarantees? Make sure this is in the test plan.
+### Graduation Criteria
 
-Consider the following in developing a version skew strategy for this
-enhancement:
-- Does this enhancement involve coordinating behavior in the control plane and
-  in the kubelet? How does an n-2 kubelet without this feature available behave
-  when this feature is used?
-- Will any other components on the node change? For example, changes to CSI,
-  CRI or CNI may require updating that component before the kubelet.
--->
-TBD
-## Production Readiness Review Questionnaire
+Alpha should provide basic functionality covered with tests described in this KEP.
```

Comment:
I could not understand the difference means of "basic functionality" and "Feature covers all important workload objects".
In my understanding, "Feature covers all important workload objects" looks out of scope of this KEP.  If it is true,  we should delete this or write detail.